### PR TITLE
86 create new structure form styling

### PR DESCRIPTION
**Pull request by DariaKozlovska**

### CODE REVIEW

**Review by Uacias — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "@storybook/html-vite": "7.6.17",
     "@types/d3": "7.4.3",
     "@types/eslint": "8.56.5",
+    "@types/jquery": "^3.5.29",
```
Comment (Uacias) [inline package.json:53]: Do we need that?

File: src/routes/app/portfolio/index.tsx [hunk 2]
```diff
@@ -265,12 +270,21 @@ export default component$(() => {
   const clickedToken = useStore({ balanceId: "", structureId: "" });
   const structureStore = useStore({ name: "" });
   const selectedWallets = useStore({ wallets: [] as any[] });
-  const isCreateNewStructureModalOpen = useSignal(false);
+  const isCreateNewStructureModalOpen = useSignal(true);
```
Comment (Uacias) [inline src/routes/app/portfolio/index.tsx:273]: imo that should be set to false by default

**Review by Uacias — CHANGES_REQUESTED** [review]

File: src/routes/app/portfolio/index.tsx [hunk 3]
```diff
@@ -5,10 +5,15 @@ import {
...
 import IconBtc from "/public/assets/icons/portfolio/btc.svg?jsx";
+import IconArrowDown from "/public/assets/icons/arrow-down.svg?jsx";
+import IconClose from "/public/assets/icons/close.svg?jsx";
 import { Group } from "~/components/groups/group";
 import {
+  $,
   component$,
   type JSXOutput,
+  QRL,
+  Signal,
```
Comment (Uacias) [inline src/routes/app/portfolio/index.tsx:16]: add type

File: src/routes/app/portfolio/index.tsx [hunk 4]
```diff
@@ -501,17 +820,52 @@ export default component$(() => {
   );
 });
 
-function parseWalletsToOptions(wallets: WalletWithBalance[]): JSXOutput[] {
+function parseWalletsToOptions(
```
Comment (Uacias) [inline src/routes/app/portfolio/index.tsx:823]: produces qwik warn

File: src/global.css [hunk 5]
```diff
@@ -186,3 +186,93 @@ body {
...
+          #f48946 40%,
+          #ee3560 60%,
+          #fa4784 80%,
+          #ff48c0 100%
+        )
+        border-box;
+    border-style: solid;
+    border-color: transparent;
+  }
+}
```
Comment (Uacias) [inline src/global.css:248]: nested css produces warn, enable nesting plugin or unnest

**Review by Uacias — CHANGES_REQUESTED** [review]

File: src/global.css (hunk 5 above)
Comment (Uacias) [reply src/global.css]: Possibly to modify in the future, once it produced qwik warn, then vanished

**Review by Uacias — APPROVED** [review]

**Review by pgajek — APPROVED** [review]

**Pull request merged**